pkg/agent/route/route_linux.go

```diff
@@ -339,6 +347,29 @@ func (c *Client) syncIPSet() error {
 		}
 	}
 
+	if c.multicastEnabled && c.networkConfig.TrafficEncapMode.SupportsEncap() {
+		if err := ipset.CreateIPSet(clusterNodeIPSet, ipset.HashIP, false); err != nil {
+			return err
+		}
+		if err := ipset.CreateIPSet(clusterNodeIP6Set, ipset.HashIP, false); err != nil {
+			return err
+		}
+		c.clusterNodeIPs.Range(func(k, _ interface{}) bool {
+			ipSetEntry := k.(string)
```

Comment:
I think `k` is podCIDR according to the code

Reply:
Updated.